### DISCUSSION TITLE
Wanting to contribute UI & GFX efforts

### DISCUSSION THREAD

**Discussion by mdomedia** [strawberrymusicplayer/strawberry#1362] [Q&A] [answered]
I love Strawberry. What's the best way to get started contributing UI and or GFX efforts to the project? Feel free to reach out directly if you can facilitate this. 

Thanks,

**Comment by jonaski** [comment, marked as answer]
Obviously there are a ton of things that could be done.
Here are some ideas, the context is probably what annoys me the most.

- Optimize the GUI for higher resolution displays such as 4K, 8K, etc.
- Optimize the analyzer for higher resolution displays, making it wider / larger.
- Improve the context, specifically so the left area can be expanded without the album cover being huge.
- Make the remaining pixmap's respect device pixel ratio, such as the collection album thumbnails.
- There should be a better way to optimize buttons for different resolutions / sizes without hard-coding this thing in the settings, a right click option would be better.
- Create a dark mode.

Please report back if you are working on something, so someone else don't start working on the same thing.

**Reply by mdomedia** [reply]
Roger that - will report back on tasks in progress if any. Thank you!
